modules/core/src/shaderlib/misc/uniform-types.ts

```diff
@@ -24,7 +24,7 @@ type NumArray12 = [
   number,
   number
 ];
-type NumArray16 = [
+export type NumArray16 = [
```

Comment:
Nit: Being averse to abbreviations, I'd vote for `NumberArray16`. And of course we will upstream it to `@math.gl/types` once this settles.

Reply:
The issue is we already have a `NumberArray` type, which is just an alias for `NumericArray`, so it would be confusing. Personally I think it might be best to retire the existing `NumberArray` type and just use `NumericArray`